Assets/MixedRealityToolkit.Tests/PlayModeTests/LoadingIndicatorTests.cs

```diff
@@ -0,0 +1,144 @@
+            while (!testTask.IsCompleted)
+            {
+                yield return null;
+            }
+
+            // clean up
+            GameObject.Destroy(progressIndicatorObject);
+            yield return null;
+        }
+
+        /// <summary>
+        /// Tests that prefab can be opened and closed at runtime.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator TestOpenCloseRotatingOrbsPrefab()
+        {
+            GameObject progressIndicatorObject;
+            IProgressIndicator progressIndicator;
+            InstantiatePrefab(progressIndicatorRotatingOrbsPrefabPath, out progressIndicatorObject, out progressIndicator);
+            Task testTask = TestOpenCloseProgressIndicatorAsync(progressIndicatorObject, progressIndicator, 3f);
+            while (!testTask.IsCompleted)
+            {
+                yield return null;
+            }
+
+            // clean up
+            GameObject.Destroy(progressIndicatorObject);
+            yield return null;
+        }
+
+        private async Task TestOpenCloseProgressIndicatorAsync(GameObject progressIndicatorObject, IProgressIndicator progressIndicator, float timeOpen = 2f)
+        {
+            // Deactivate the progress indicator
+            progressIndicatorObject.SetActive(false);
+
+            // Make sure it's closed
+            Assert.True(progressIndicator.State == ProgressIndicatorState.Closed, "Progress indicator was not in correct state on startup: " + progressIndicator.State);
+
+            // Make sure we can set progress and message
+            progressIndicator.Progress = 0;
+            progressIndicator.Message = "Progress Test";
+
+            // Wait for it to open
+            await progressIndicator.OpenAsync();
+
+            // Make sure it's actually open
+            Assert.True(progressIndicator.State == ProgressIndicatorState.Open, "Progress indicator was not open after open async call: " + progressIndicator.State);
+
+            // Make sure we can set its progress and message while open
+            // Also make sure we can set progress to a value greater than 1 without blowing anything up
+            float timeStarted = Time.time;
+            while (Time.time < timeStarted + timeOpen)
+            {
+                progressIndicator.Progress = Time.time - timeStarted;
+                progressIndicator.Message = "Current Time: " + Time.time;
+                await Task.Yield();
+            }
+
+            // Wait for it to close
+            await progressIndicator.CloseAsync();
+
+            // Make sure it's actually closed
+            Assert.True(progressIndicator.State == ProgressIndicatorState.Closed, "Progress indicator was not closed after close async call: " + progressIndicator.State);
+        }
+
+        [TearDown]
+        public void ShutdownMrtk()
+        {
+            //TestUtilities.ShutdownMixedRealityToolkit();
```

Comment:
>TestUtilities [](start = 14, length = 13)

Delete this line (and probably the TearDown function as well because it'll be empty)